python-project-template/.github/workflows/{% if preferred_linter != 'none' %}pre-commit-ci.yml{% endif %}.jinja

```diff
@@ -0,0 +1,34 @@
+# This workflow runs pre-commit hooks on pull requests to enforce coding style.
+# To ensure correct configuration, please refer to:
+#   https://lincc-ppt.readthedocs.io/en/latest/practices/ci_precommit.html
+
+name: Run pre-commit hooks
+
+on:
+  pull_request:
+
+jobs:
+  pre-commit-ci:
+    runs-on: ubuntu-latest
+    env: 
+      SKIP: "check-lincc-frameworks-template-version,pytest-check,no-commit-to-branch,validate-pyproject,check-added-large-files,sphinx-build"
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0 
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.10'
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
+        python -m pip install --upgrade pip
+        pip install .
+        pip install .[dev]
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+    - uses: pre-commit/action@v3.0.0
+      with:
+        extra_args: --from-ref {% raw %}${{ github.event.pull_request.base.sha }}{% endraw %} --to-ref {% raw %}${{ github.event.pull_request.head.sha }}{% endraw %}
+    - uses: pre-commit-ci/lite-action@v1.0.1
+      if: always()
```

Comment:
Is the `if: always()` necessary here? I don't know enough to know.

Reply:
I've followed this [recommendation](https://pre-commit.ci/lite) from the **pre-commit.ci lite** webpage.